index.html

```diff
@@ -4,14 +4,29 @@
     <meta charset="utf-8">
     <title>Code Coop</title>
     <link rel="stylesheet" href="assets/css/index.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,700" rel="stylesheet">
```

Comment:
💯Good call adding a nice font.

Reply:
Cheers! Thought it was a nice little extra touch.